rose/server/player.py

```diff
@@ -12,6 +12,10 @@ def __init__(self, name, car, lane):
         self.action = None
         self.response_time = None
         self.score = None
+        self.penguin_collected = 0
+        self.water_collected = 0
+        self.crack_collected = 0
+        self.collision_count = 0
```

Comment:
```suggestion
        self.penguin_collected = None
        self.water_collected = None
        self.crack_collected = None
        self.collision_count = None
```

the convention is to init to None and then use reset to set to 0